<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulesWidget</class>
 <widget class="QWidget" name="SchedulesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 36px;
color: black;</string>
   </property>
   <property name="text">
    <string>Horarios</string>
   </property>
  </widget>
  <widget class="QWidget" name="toggleContainer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="maestroButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #5c7c9e;
color: white;
border-radius: 5px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Maestro</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="grupoButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #5c7c9e;
color: white;
border-radius: 5px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Grupo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e0e0e0;
border-radius: 20px;
padding-left: 15px;
font-size: 14px;
color: #606060;</string>
   </property>
   <property name="placeholderText">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>761</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="itemFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #5c7c9e;
border-radius: 5px;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="styleSheet">
           <string notr="true">color: white; font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Nombreplaceholder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line1">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="subjectsLabel">
          <property name="styleSheet">
           <string notr="true">color: white; font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Materias</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hoursLabel">
          <property name="styleSheet">
           <string notr="true">color: white; font-weight: bold;</string>
          </property>
          <property name="text">
           <string>horas</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
